Product/Production/Deploy/ear/weblogic/pom.xml

```diff
@@ -19,11 +19,6 @@
         </dependency>
         <dependency>
             <groupId>org.apache.geronimo.specs</groupId>
-            <artifactId>geronimo-jaxws_2.2_spec</artifactId>
```

Comment:
This jar needs to be removed as per OWASP dependency documentation.
https://cgiinterop.atlassian.net/wiki/display/FCPT/OWASP+Dependency+Check